lodash/lodash.d.ts

```diff
@@ -9928,11 +9928,13 @@ declare module _ {
     interface LoDashStatic {
         /**
          * Checks if value is a finite primitive number.
+         *
          * Note: This method is based on Number.isFinite.
+         *
          * @param value The value to check.
          * @return Returns true if value is a finite number, else false.
-         **/
-        isFinite(value?: any): value is number;
+         */
+        isFinite(value?: any): boolean;
```

Comment:
@vvakame There is one case where it is not works http://goo.gl/Weq5aV . Am I wrong?